libs/metrics/top_level_metrics.py

```diff
@@ -292,6 +292,13 @@ def calculate_contact_tracers(
 
 
 def calculate_covid_patient_ratio(data: pd.DataFrame, region: Region):
+    # If we have precomputed data, use it.
+    if (
+        CommonFields.BEDS_WITH_COVID_PATIENTS_RATIO_HSA in data.columns
```

Comment:
The `CommonFields.BEDS_WITH_COVID_PATIENTS_RATIO_HSA in data.columns` is here so I don't have to update tests and the `test-combined-wide-dates.csv` file... which is maybe hacky?  Not sure.

Reply:
This is fine with me--it also means we might create community levels for locations that the CDC doesn't cover?

Reply:
Yeah.  I think in that case the column will exist (it basically only doesn't exist in tests) but won't have any data, and we will indeed calculate the community level ourselves, which I think is fine / good?